package.json

```diff
@@ -14,13 +14,17 @@
   "contributors": [
     "Shaangor (https://github.com/Shaangor)",
     "Carl-Hugo Marcotte (https://github.com/Carl-Hugo)",
-    "Layton B (https://github.com/LaytonGB)"
+    "Layton B (https://github.com/LaytonGB)",
+    "Adam D (https://github.com/laquasicinque"
   ],
   "license": "MIT",
   "bugs": {
     "url": "https://github.com/Carl-Hugo/types-roll20/issues"
   },
   "homepage": "https://github.com/Carl-Hugo/types-roll20#readme",
-  "devDependencies": {},
-  "dependencies": {}
+  "devDependencies": {
+    "@types/underscore": "^1.10.24",
+    "typescript": "^4.1.0-beta"
```

Comment:
We should most likely wait for a release version before merging the PR, don't you think?

Reply:
Probably worth waiting, alternatively, if you can work out how to do it, provide like a beta version that people can opt into for version 4.1 types before 4.1 is out